README-install.md

```diff
@@ -72,6 +72,51 @@ file like `/etc/sudoers.d/999-cloud-init-requiretty` with the following contents
     # cat /etc/sudoers.d/999-cloud-init-requiretty
     Defaults !requiretty
 
+Persistent Storage
+------------------
+
+Elasticsearch, Fluentd, and mux all require persistent storage - Elasticsearch
+for the database, and Fluentd and mux for the file buffering.  Fluentd and mux
+will use `/var/lib/fluentd` and `/var/log/fluentd`, respectively, by default.
```

Comment:
These paths are hostPath when hostmount is chosen.  (fluentd has no other option, while mux has 3 options - hostmount, pvc, and emptydir.  The mountPath (path in the container) is always /var/lib/fluentd.
```
    [fluentd.j2]
        hostPath:
          path: "/var/lib/fluentd"
    [mux.j2]
        hostPath:
          path: "/var/log/fluentd"
```
If we don't want to advertise pvc and emptydir, please ignore this comment.  But it might be helpful to mention the mountPath is always /var/lib/fluentd.